### PR TITLE
Masterbots/UI fixes

### PR DESCRIPTION
**Pull request by AnoukRImola**
In this PR, the following changes were made:

Desktop:

- The class was modified so that the question window remains static regardless of the number of questions:

image

Mobile:

-  The sticky was fixed.
-  A scroll was added for when the questions are too long.
-  The right margin was removed as instructed.
-  The Sidebar Icon was modified to appear only when navigating the 'chat' route.
image
image

## Summary by CodeRabbit

- **Style Updates**
  - Standardized CSS class names for consistency across multiple components, improving maintainability and readability.
  - Updated width, height, and size class names to new conventions (`size-4`, `size-6`, `size-7`, `size-9`) for several elements.
  - Adjusted maximum width classes to utilize screen size breakpoints (e.g., `max-w-screen-lg`).

- **Refactor**
  - Wrapped `SidebarToggle` in a `React.Suspense` component for improved code splitting and loading states.
  - Introduced `SidebarToggleWrap` to conditionally render based on the current pathname.

### CODE REVIEW

File: apps/masterbots.ai/components/browse-chat-message-list.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export function BrowseChatMessageList({
           <></>
```
[the remark this answers is not in the document]
Comment (AndlerRL) [reply apps/masterbots.ai/components/browse-chat-message-list.tsx:56]: Please clean this line.

File: apps/masterbots.ai/components/thread-popup.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ export function ThreadPopup({ className }: { className?: string }) {
             )}
           </div>
           <button
-            className="absolute right-[32px] top-[50%] translate-y-[-50%] "
+            className="absolute right-[32px] top-1/2 translate-y-1/2 "
             onClick={onClose}
           >
             <IconClose />
```
[the remark this answers is not in the document]
Comment (AndlerRL) [reply apps/masterbots.ai/components/thread-popup.tsx:93]: Good suggestion. @AnoukRImola , can you apply the update? I tried with coderabbitai but it throws errors.

**Review by AndlerRL — APPROVED** [review]
LGTM! I notice only 1 tweak and a clean up. After that, it will be ready for a merge!

**Comment by AndlerRL** [comment]
@AnoukRImola -- please handle the conflict with the changes 🙏

**Comment by AnoukRImola** [comment]
@AndlerRL ,I cleaned up, updated more Tailwind classes, fixed the conflict with the bun.lockb file, and updated the input texts to align to the left.

image

**Comment by AndlerRL** [comment]
@AnoukRImola -- Thanks for taking care of the feedbacks! I see though 2 left from coderabbitai. Also, I noticed a build error on the last commit -> https://vercel.com/bitcash/masterbots/2wHnPgDMxH1pFtbi9rXL9b5Qdmrn?filter=errors can you check that out?

**Comment by AndlerRL** [comment]
[[FE] UI/UX Tweaks Hotfixes - Mobile Threads + Dialog](https://trello.com/c/ZOU7im9t/177-fe-ui-ux-tweaks-hotfixes-mobile-threads-dialog)

**Pull request merged**